tutorial/deploy_jaas_microk8s.rst

```diff
@@ -259,7 +259,11 @@ Run the following command to unseal Vault and export the unseal token and root k
 
 Now run ``juju status`` again and confirm your Vault unit is in an active state.
 
-Finally, save the root token and unseal key to disk for use later. The unseal key is especially important if your PC is restarted as Vault will become resealed and the unseal key will be needed again.
+Finally, save the root token and unseal key to disk for use later.
+
+.. note::
+
+    The unseal key is especially important. If your PC is restarted, Vault will become resealed and the unseal key will be needed again.
```

Comment:
If any of vault pods gets recreated, in fact.. then that unit gets sealed and needs to be unsealed manually.